src/config/src/config.erl

```diff
@@ -212,23 +212,23 @@ delete(Section, Key, Persist, Reason) when is_list(Section), is_list(Key) ->
     ).
 
 features() ->
-    application:get_env(config, enabled_features, []).
+    Map = persistent_term:get({?MODULE, ?FEATURES}, #{}),
+    lists:sort(maps:keys(Map)).
```

Comment:
how much does the lists:sort each time impact the performance gain?

Reply:
It looks minimal about an extra 1 microsecond at most with the default features list.

```
element(1, timer:tc(fun() -> (fun F(0) -> ok; F(N) -> config:features(), F(N-1) end)(1000000) end)) / 1000000.
1.014499
```

`config:features()` would be used from the Welcome endpoint only, specific feature checks would be quicker as we don't need to sort a list in that case.

Reply:
It's on-par or faster than the previous approach before the persist term update:

```
 element(1, timer:tc(fun() -> (fun F(0) -> ok; F(N) -> config:features(), F(N-1) end)(1000000) end)) / 1000000.
1.201835
```